modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ObjectEntryDisplayContext.java

```diff
@@ -595,17 +595,12 @@ private DDMForm _getDDMForm(ObjectLayoutTab objectLayoutTab)
 			}
 		}
 
-		List<ObjectField> objectFields =
-			_objectFieldLocalService.getObjectFields(
-				objectDefinition.getObjectDefinitionId());
-
 		if (objectLayoutTab == null) {
-			for (ObjectField objectField : objectFields) {
-				if (!_isActive(objectField)) {
-					continue;
-				}
+			for (ObjectField objectField :
+					_objectFieldLocalService.getCustomObjectFields(
+						objectDefinition.getObjectDefinitionId(), true)) {
```

Comment:
Don't you mean the second parameter to be false? So the method will return only the custom fields and exclude the system fields.

I was reviewing the method from the persistence:
`_objectFieldPersistence.findByODI_NotS(objectFieldId, system);`

If your implementation is right and sending true to parameter "system" means that we are getting the custom fields, I would think in renaming the parameter or deny (!system) when calling the persistence method. I'm saying that because I got confused here :confused: sorry about it!

We can discuss it whenever you want :+1: